### DISCUSSION TITLE
【上海线下 2022.11.12】Move 交流会

### DISCUSSION THREAD

**Discussion by Demian101** [NonceGeek/Move-Camp#199] [Tricks]
📌  [ 上海   Builder 看过来] ~  
我打算这周搞一个 move 语言的线下交流小会，不知道大家有没有兴趣，可以交流下：
1. 语言学习问题
2. 一些项目的 idea 的分享（if 
3. 行业资源分享

（ 会后有价值的信息我也会整理到 github/materials

**Comment by Demian101** [comment]
![image](https://user-images.githubusercontent.com/33189338/200173046-8b30651c-005c-4599-bcc7-df880c85d160.png)

**Comment by leeduckgo** [comment]
So Cool!
